README.md

```diff
@@ -1,38 +1,68 @@
-ChatGPT Exporter
-==========================
-
-![license](https://img.shields.io/github/license/pionxzh/chatgpt-exporter?color=red)
-[![](https://img.shields.io/static/v1?label=%20&message=GreasyFork&style=flat-square&labelColor=7B0000&color=960000&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH3ggEBCQHM3fXsAAAAVdJREFUOMudkz2qwkAUhc/goBaGJBgUtBCZyj0ILkpwAW7Bws4yO3AHLiCtEFD8KVREkoiFxZzX5A2KGfN4F04zMN+ce+5c4LMUgDmANYBnrnV+plBSi+FwyHq9TgA2LQpvCiEiABwMBtzv95RSfoNEHy8DYBzHrNVqVEr9BWKcqNFoxF6vx3a7zc1mYyC73a4MogBg7vs+z+czO50OW60Wt9stK5UKp9Mpj8cjq9WqDTBHnjAdxzGQZrPJw+HA31oulzbAWgLoA0CWZVBKIY5jzGYzdLtdE9DlcrFNrY98zobqOA6TJKHW2jg4nU5sNBpFDp6mhVe5rsvVasUwDHm9Xqm15u12o+/7Hy0gD8KatOd5vN/v1FozTVN6nkchxFuI6hsAAIMg4OPxMJCXdtTbR7JJCMEgCJhlGUlyPB4XfumozInrupxMJpRSRtZlKoNYl+m/6/wDuWAjtPfsQuwAAAAASUVORK5CYII=)](https://greasyfork.org/scripts/456055-chatgpt-exporter)
+<h1 align="center">ChatGPT Exporter</h1>
+
+<div align="center">
+    
+## A GreasyFork script to export the chat history of [ChatGPT](https://chatgpt.com/)
+
+[![license][license-image]][license-url]
+[![release][release-image]][release-url]
+[![last commit][last-commit-image]][last-commit-url]
+[![GreasyFork][GreasyFork-image]][GreasyFork-url]    
+    
+[license-image]: https://img.shields.io/github/license/pionxzh/chatgpt-exporter?color=red
+[license-url]: https://github.com/pionxzh/chatgpt-exporter/blob/master/LICENSE
+[release-image]: https://img.shields.io/github/v/release/pionxzh/chatgpt-exporter?color=blue
+[release-url]: https://github.com/pionxzh/chatgpt-exporter/releases/latest
+[last-commit-image]: https://img.shields.io/github/last-commit/pionxzh/chatgpt-exporter?label=last%20commit
+[last-commit-url]: https://github.com/pionxzh/chatgpt-exporter/commits
+[GreasyFork-image]: https://img.shields.io/static/v1?label=%20&message=GreasyFork&style=flat-square&labelColor=7B0000&color=960000&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH3ggEBCQHM3fXsAAAAVdJREFUOMudkz2qwkAUhc/goBaGJBgUtBCZyj0ILkpwAW7Bws4yO3AHLiCtEFD8KVREkoiFxZzX5A2KGfN4F04zMN+ce+5c4LMUgDmANYBnrnV+plBSi+FwyHq9TgA2LQpvCiEiABwMBtzv95RSfoNEHy8DYBzHrNVqVEr9BWKcqNFoxF6vx3a7zc1mYyC73a4MogBg7vs+z+czO50OW60Wt9stK5UKp9Mpj8cjq9WqDTBHnjAdxzGQZrPJw+HA31oulzbAWgLoA0CWZVBKIY5jzGYzdLtdE9DlcrFNrY98zobqOA6TJKHW2jg4nU5sNBpFDp6mhVe5rsvVasUwDHm9Xqm15u12o+/7Hy0gD8KatOd5vN/v1FozTVN6nkchxFuI6hsAAIMg4OPxMJCXdtTbR7JJCMEgCJhlGUlyPB4XfumozInrupxMJpRSRtZlKoNYl+m/6/wDuWAjtPfsQuwAAAAASUVORK5CYII=
+[GreasyFork-url]: https://greasyfork.org/scripts/456055-chatgpt-exporter
+    
+English 🇺🇸 &nbsp;&nbsp;|&nbsp;&nbsp; [Indonesia 🇮🇩](./README_ID.md) &nbsp;&nbsp;|&nbsp;&nbsp; [한국어 🇰🇷](./README_KR.md) &nbsp;&nbsp;|&nbsp;&nbsp; [Türkçe 🇹🇷](./README_TR.md)
```

Comment:
Let's keep it no country flag

Reply:
> Let's keep it no country flag

I've deleted it, if anything needs to be fixed just let me know

Reply:
Thanks. The reason for this is that I want to decouple the relationship between language and country.